prow/cluster/05-tiller.yaml

```diff
@@ -0,0 +1,92 @@
+# Generated by Helm 1.11
```

Comment:
Do we really want to install helm/tiller this way? Wouldn't it be better to use `helm init` from the `install-prow.sh` script?

Reply:
We want to have fixed version of tiller in cluster. Helm init will install same version as client.